Update assessment flow styling

ericwbailey opened the pull request:
<!-- Required: read https://github.com/Path-Check/covid-safe-paths/wiki/Pull-Request-Best-Practices for recommended best practices before opening your first pull request.  PR's raised not following those guidelines will require rework, so you might as well start off right -->

#### Description:

This PR:

- Matches typography and spacing with the most current Figma design
- Uses a standardized header for assessment content
- Updates checkboxes to use squared inputs
- Matches background and border colors with the most current Figma design
- Aligns checkboxes with the top of the input label
- Updates the disabled button styling with the most current Figma design
- Updates spacing values to use spacing constants
- Adds assessment question instructions based on question type (Note: we'll need to update this to be able to be localized)

[Relevant Trello ticket](https://trello.com/c/iPJgng1e).

#### Linked issues:

<b>Trello:</b>

- [As a user, I want to be able to read assessment instructions in my local and preferred language](https://trello.com/c/BnjWcocE)
- [As a user, I want the self-assessment flow to be consistent with the rest of the application and be a smoother user experience](https://trello.com/c/31THPE8p)

#### Screenshots:

<details>
 <summary>Before</summary>

 <b>Radio</b>
 <img width="574" alt="ScreenCapture at Fri Jun 26 11:33:02 EDT 2020" src="https://user-images.githubusercontent.com/634191/85877216-ce301a80-b7a4-11ea-8bab-1decb293a846.png">

 <b>Checkbox</b>
 <img alt="Simulator Screen Shot - iPhone 11 - 2020-06-26 at 11 33 32" src="https://user-images.githubusercontent.com/634191/85877246-d6885580-b7a4-11ea-92ef-727fc9495a68.png">
</details>

<details>
 <summary>After</summary>

 <b>Radio</b>
 <img width="574" alt="ScreenCapture at Fri Jun 26 11:38:32 EDT 2020" src="https://user-images.githubusercontent.com/634191/85877388-151e1000-b7a5-11ea-94ba-fbad93cdb776.png">

 <b>Checkbox</b>
 <img width="574" alt="ScreenCapture at Fri Jun 26 11:38:47 EDT 2020" src="https://user-images.githubusercontent.com/634191/85877415-1f400e80-b7a5-11ea-900f-fa3ba8d543c9.png">

</details>

#### How to test:

Navigate through the assessment flow.


Co-authored-by: achen178 <achen178@gmail.com>

ericwbailey committed:
WIP

ericwbailey committed:
Update checkbox style

ericwbailey committed:
Tweak input border width and color

ericwbailey committed:
Update assessmentInstruction styling

ericwbailey committed:
Add radioBorder and checkboxBorder styles

ericwbailey committed:
Add instruction logic, update spacing/typography

ericwbailey committed:
Merge branch 'develop' into eb-ac-update-header-titles

* develop:
  Refactor, update styling, update content for about page (#1116)
  Add dev exposure notifications entitlements. (#1117)

# Conflicts:
#	app/styles/colors.ts

ericwbailey committed:
Formatting

ericwbailey committed:
Update snapshot

johnschoeman commented on app/components/Option/OptionSelect.js:
```diff
@@ -22,40 +22,51 @@ export function OptionSelect({
 }
 
-const styles = StyleSheet.create({
+const inputStyle = StyleSheet.create({
```
What do you think about putting this in the existing stylesheet? Not sure we need two here. something thing:

```
const styles = StyleSheet.create({
  input: {
      alignItems: 'center',
      ...
   },
  indicatorRadio: {
    ...inputStyle.indicator,
    borderRadius: Spacing.medium * 2,
  },
```

ericwbailey replied:
Sure thing, I can give that a shot.

ericwbailey replied:
We're getting an error of `TypeError: undefined is not an object (evaluating styles.indicator)` when we try this. 

The reason it was separate initially was to avoid this error. Is there a way to extend a style within `styles`? In Sass it'd be akin to a mixin to coordinate shared properties.

ericwbailey replied:
Scratch that, we wound up migrating it into `forms.ts`.

johnschoeman commented on app/components/Option/index.js:
```diff
@@ -27,18 +27,18 @@ const styles = StyleSheet.create({
-    paddingHorizontal: 10,
-    paddingVertical: 20,
+    borderRadius: Spacing.xxSmall,
+    borderWidth: 2,
```
This *might* want to live in the design system. Do you see this border being a repeated pattern?

```suggestion
    ...Outlines.secondaryBoarder
```

ericwbailey replied:
Funny, @achen178 and I discussed this, but we thought it would be a little outside of this PR's scope. Cool to introduce that as a separate PR?

johnschoeman replied:
> Funny, @achen178 and I discussed this, but we thought it would be a little outside of this PR's scope. Cool to introduce that as a separate PR?

Totally, it’s your pr!

johnschoeman reviewed (approved):
Overall looks great! a few thoughts.

ericwbailey committed:
Add inputIndicator to centralize input styling

This moves inputStyle.indicator into one common indicator style in styling, so that all checkboxes and radios can draw from the same formatting source.

ericwbailey committed:
Merge branch 'develop' into eb-ac-update-header-titles

* develop:
  Lokalise latest june 26 (#1127)

ericwbailey committed:
Format

ericwbailey committed:
Merge branch 'develop' into eb-ac-update-header-titles

* develop:
  A Few Minor Updates (#1121)
  Correct EN Permissions State on App Launch (#1126)
  Add strategy content to tracing strategy
  Fix unused style import (#1111)
